### PR TITLE
Update table prefix to `wp_pc_`

### LINKED ISSUE #121: Table prefix
### Is your enhancement related to a problem? Please describe.

When testing up a the test tables to test the site, it does not use the prefix defined. It hardcode table prefix. There maybe a problem for a number of reason. 

1. Multisite, there tables are prefixed. 
2. Hosting companies that use one database from my sites. Two sites may this plugin installed and use the table. 
3. Hosting companies that set permissions based on the table prefix. 

I would recommend, post fixing the prefix. So `wp_pc_` with `wp_` in the base prefix. 

### Designs

_No response_

### Describe alternatives you've considered

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by jjgrainger**
Closes #121

### CODE REVIEW

**Comment by spacedmonkey** [comment]
@jjgrainger I think the original ticket's meaning was unclear. Developers / hosts can config there database config. See this [line](https://github.com/WordPress/wordpress-develop/blob/0276bfdd6d714810542866adf4325619b873db67/wp-config-sample.php#L68) 

So it should use the global $table_prefix.

File: includes/Checker/Abstract_Check_Runner.php [hunk 1]
```diff
@@ -154,7 +154,7 @@ public function prepare() {
 
 			// Set the database prefix to use the demo tables.
 			global $wpdb;
```
Comment (spacedmonkey) [inline includes/Checker/Abstract_Check_Runner.php]:
```suggestion
			global $wpdb, $table_prefix;
```

File: includes/Checker/Abstract_Check_Runner.php [hunk 2]
```diff
@@ -154,7 +154,7 @@ public function prepare() {
...
-			$old_prefix = $wpdb->set_prefix( 'wppc_' );
+			$old_prefix = $wpdb->set_prefix( 'wp_pc_' );
```
Comment (spacedmonkey) [inline includes/Checker/Abstract_Check_Runner.php]:
```suggestion
			$old_prefix = $wpdb->set_prefix( $table_prefix . 'pc_' );
```

File: includes/Admin/Admin_AJAX.php [hunk 3]
```diff
@@ -123,12 +123,12 @@ public function clean_up_environment() {
 		}
 
 		// Set the new prefix.
-		$old_prefix = $wpdb->set_prefix( 'wppc_' );
+		$old_prefix = $wpdb->set_prefix( 'wp_pc_' );
```
Comment (spacedmonkey) [inline includes/Admin/Admin_AJAX.php]: See https://github.com/10up/plugin-check/pull/146/files#r1168917663

File: includes/Admin/Admin_AJAX.php [hunk 4]
```diff
@@ -123,12 +123,12 @@ public function clean_up_environment() {
...
 
 		$message = __( 'Runtime environment was not prepared, cleanup was not run.', 'plugin-check' );
 
 		// Test if the runtime environment tables exist.
-		if ( 'wppc_posts' === $wpdb->get_var( "SHOW TABLES LIKE 'wppc_posts'" ) || defined( 'WP_PLUGIN_CHECK_OBJECT_CACHE_DROPIN_VERSION' ) ) {
+		if ( 'wp_pc_posts' === $wpdb->get_var( "SHOW TABLES LIKE 'wp_pc_posts'" ) || defined( 'WP_PLUGIN_CHECK_OBJECT_CACHE_DROPIN_VERSION' ) ) {
```
Comment (spacedmonkey) [inline includes/Admin/Admin_AJAX.php]: This same `wp_pc_posts` is used twice. It should be a variable.

File: includes/Checker/Runtime_Environment_Setup.php [hunk 5]
```diff
@@ -25,10 +25,10 @@ public function setup() {
 		require_once ABSPATH . '/wp-admin/includes/upgrade.php';
 
 		// Set the new prefix.
-		$old_prefix = $wpdb->set_prefix( 'wppc_' );
+		$old_prefix = $wpdb->set_prefix( 'wp_pc_' );
```
Comment (spacedmonkey) [inline includes/Checker/Runtime_Environment_Setup.php]: Should be a variable, as `wp_pc_` is repeated a lot.

**Review by spacedmonkey — CHANGES_REQUESTED** [review]
I think the point of this PR is lost. Let's use the table_prefix global everywhere.

**Review by felixarntz — COMMENTED** [review]
Nothing to add here from my end to @spacedmonkey's feedback.

File: includes/Admin/Admin_AJAX.php (hunk 4 above)
Comment (spacedmonkey) [inline includes/Admin/Admin_AJAX.php]:
```suggestion
		if ( $wpdb->posts === $wpdb->get_var( $wpdb->prepare( "SHOW TABLES LIKE %s",  $wpdb->posts ) ) || defined( 'WP_PLUGIN_CHECK_OBJECT_CACHE_DROPIN_VERSION' ) ) {
```

File: includes/Checker/Runtime_Environment_Setup.php [hunk 6]
```diff
@@ -25,10 +25,10 @@ public function setup() {
...
 
 		// Create and populate the test database tables if they do not exist.
-		if ( 'wppc_posts' !== $wpdb->get_var( "SHOW TABLES LIKE 'wppc_posts'" ) ) {
+		if ( 'wp_pc_posts' !== $wpdb->get_var( "SHOW TABLES LIKE 'wp_pc_posts'" ) ) {
```
Comment (spacedmonkey) [inline includes/Checker/Runtime_Environment_Setup.php]:
```suggestion
		if ( $wpdb->posts === $wpdb->get_var( $wpdb->prepare( "SHOW TABLES LIKE %s",  $wpdb->posts ) ) ) {
```

**Comment by jjgrainger** [comment]
Thnaks @spacedmonkey I've updated this based on your feedback and is ready for review.

**Review by felixarntz — APPROVED** [review]
Thanks for the updates @jjgrainger!

File: includes/Checker/Runtime_Environment_Setup.php [hunk 7]
```diff
@@ -20,15 +20,15 @@ class Runtime_Environment_Setup {
...
 
 		require_once ABSPATH . '/wp-admin/includes/upgrade.php';
 
 		// Set the new prefix.
-		$old_prefix = $wpdb->set_prefix( 'wppc_' );
+		$old_prefix = $wpdb->set_prefix( $table_prefix . 'pc_' );
 
 		// Create and populate the test database tables if they do not exist.
-		if ( 'wppc_posts' !== $wpdb->get_var( "SHOW TABLES LIKE 'wppc_posts'" ) ) {
+		if ( $wpdb->posts === $wpdb->get_var( $wpdb->prepare( 'SHOW TABLES LIKE %s', $wpdb->posts ) ) ) {
```
Comment (spacedmonkey) [inline includes/Checker/Runtime_Environment_Setup.php]:
```suggestion
		if ( $wpdb->posts !== $wpdb->get_var( $wpdb->prepare( 'SHOW TABLES LIKE %s', $wpdb->posts ) ) ) {
```

**Review by spacedmonkey — APPROVED** [review]
Looking good.

**Pull request merged**